src/util/v4/test/v4.cc

```diff
@@ -222,22 +222,22 @@ TEST_CASE("TEST_CASE_load_4x1_tr", "[v4]") {
 } // TEST_CASE
 
 TEST_CASE("TEST_CASE_load_4x2_tr", "[v4]") {
-  DECLARE_ALIGNED_ARRAY( int, 16, mem, 16 );
+  DECLARE_ALIGNED_ARRAY( int, 16, mem, 32 );
   v4int a0, a1, a2, a3;
   int i;
-  for( i=0; i<16; i++ ) mem[i] = i;
-  load_4x2_tr(mem,  mem+4,mem+8, mem+12,a0,a1);
-  load_4x2_tr(mem+2,mem+6,mem+10,mem+14,a2,a3);
-  for( i=0; i<16; i++ ) if( mem[i]!=i ) break;
+  for( i=0; i<32; i++ ) mem[i] = i;
```

Comment:
Is there any performance implication to this increase in size?

Reply:
There is a correctness issue that showed up with ARM.  The second call to load_4x2_tr was trying to access memory beyond the allocated length of mem.  Other compilers like Intel and GNU were not complaining.  ARM did and the test was failing.  Making mem longer fixed the problem.